src/main/webapp/index.html

```diff
@@ -5,40 +5,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  
     <title>GPostCard</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/env.css">
+    <link rel="stylesheet" href="style/style.css">
```

Comment:
already declared

Reply:
these are different. Mine and Andrei's. Different folders

Reply:
We will use the same directory, but probably after tomorrow

Reply:
sorry, this need to be fixed before, can't really approve a PR with this inconsistency